config/routes.rb

```diff
@@ -9,6 +9,10 @@
     get "demo_partials/new"
     get "demo_partials/edit"
 
+    get "signup",   to: "users#new",            as: :signup
+    post "signup",  to: "users#create"        
+
     resources :products
+    resources :users, only: %i[new create show]
```

Comment:
```suggestion
    resources :users, only: %i[show]
```
2 action này thừa, đã có router trên signup r